ui/app/templates/components/client-node-row.hbs

```diff
@@ -6,20 +6,17 @@
   {{/if}}
 </td>
 <td data-test-client-id>{{#link-to "clients.client" node.id class="is-primary"}}{{node.shortId}}{{/link-to}}</td>
-<td data-test-client-name class="is-200px is-truncatable" title="{{node.name}}">{{node.name}}</td>
-<td data-test-client-status>{{node.status}}</td>
-<td data-test-client-drain>
-  {{#if node.isDraining}}
-    <span class="status-text is-info">true</span>
-  {{else}}
-    false
-  {{/if}}
+<td class="is-200px is-truncatable is-hover-overflowable" title="{{node.name}}">
```

Comment:
I think the `is-200px` modifier should either be removed or replaced with [another width modifier](https://github.com/hashicorp/nomad/blob/master/ui/app/styles/core/table.scss#L65-L112) so more name is shown at rest.

This will impact how much room ipv6 addresses have, but I think it's more common to have a long name than an ipv6 address. Maybe we can use the `is-hover-overflowable` pattern on the name column too?

Reply:
I changed it to 300 and applied it to addresses as well. 🎉